tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
No. the recursive relationship will not be displayed, but the rest of the case types will show up.

tested with (abbreviated):

```
            subsets: [{                                                                                                                                                                                     
                id: "grandparent",                                                                                                                                                                          
                name: "grandparent (household)",                                                                                                                                                            
            }, {                                                                                                                                                                                            
                id: "parent",                                                                                                                                                                               
                name: "parent (mother)",                                                                                                                                                                    
                related: {                                                                                                                                                                                  
                    parent: "grandparent",                                                                                                                                                                  
                    "first-child": "case",                                                                                                                                                                  
                }                                                                                                                                                                                           
            }, {                                                                                                                                                                                            
                id: "case",                                                                                                                                                                                 
                name: "case (child)",                                                                                                                                                                       
                related: {                                                                                                                                                                                  
                    parent: "parent",                                                                                                                                                                       
                },                                                                                                                                                                                          
            }]                                                                                                                                                                                              

```